docs/ibc/overview.md

```diff
@@ -10,131 +10,147 @@ Learn what IBC is, its components and use cases. {synopsis}
-optionally acknowledgement) structs as well as how to encode/decode them, and implement the
-`IBCModule` interface. Below is a more detailed explanation of how to write an IBC application
+To have your module interact over IBC you must: 
+
+- Bind to a port or ports
+- Define your own packet data
+- Define optional acknowledgement structs
+- Know how to encode and decode the packet data
+- Implement the `IBCModule` interface
+
+Read on for a detailed explanation of how to write an IBC application
 module correctly.
 
 ## Components Overview
 
 ### [Clients](https://github.com/cosmos/ibc-go/blob/main/modules/core/02-client)
 
-IBC Clients are light clients (identified by a unique client-id) that track the consensus states of
+IBC clients are light clients that are identified by a unique client-id. IBC clients track the consensus states of
 other blockchains, along with the proof spec necessary to properly verify proofs against the
-client's consensus state. A client may be associated with any number of connections to the counterparty
-chain. The supported IBC clients are:
+client's consensus state. A client can be associated with any number of connections to the counterparty
+chain. 
+
+The supported IBC clients are:
 
-* [Solo Machine light client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/06-solomachine): devices such as phones, browsers, or laptops.
-* [Tendermint light client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint): The default for SDK-based chains,
+* [Solo Machine light client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/06-solomachine): Devices such as phones, browsers, or laptops.
+* [Tendermint light client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/07-tendermint): The default for Cosmos SDK-based chains.
 * [Localhost (loopback) client](https://github.com/cosmos/ibc-go/blob/main/modules/light-clients/09-localhost): Useful for
-testing, simulation and relaying packets to modules on the same application.
+testing, simulation, and relaying packets to modules on the same application.
 
 ### [Connections](https://github.com/cosmos/ibc-go/blob/main/modules/core/03-connection)
 
-Connections encapsulate two `ConnectionEnd` objects on two seperate blockchains. Each
-`ConnectionEnd` is associated with a client of the other blockchain (ie counterparty blockchain).
+Connections encapsulate two `ConnectionEnd` objects on two separate blockchains. Each
+`ConnectionEnd` is associated with a client of the other blockchain (for example, the counterparty blockchain).
 The connection handshake is responsible for verifying that the light clients on each chain are
 correct for their respective counterparties. Connections, once established, are responsible for
-facilitation all cross-chain verification of IBC state. A connection may be associated with any
+facilitation all cross-chain verification of IBC state. A connection can be associated with any
```

Comment:
I changed all instances of may to can
per Google developer documentation style guide advice for using can https://developers.google.com/style/word-list#letter-c 
and  may https://developers.google.com/style/word-list#may

key point

> may
> In general, reserve _may_ for official policy or legal considerations. To convey possibility, use _can_ or _might_ instead. To convey permission, use _can_ instead. See also can, might, and should.

Reply:
Should this not read as: "Connections, once established, are responsible for **facilitating** all cross-chain verification of IBC state"?

Another small point which I am unsure of but I assumed previously - I thought the use of `may` here was as per [RFC-2119](https://datatracker.ietf.org/doc/html/rfc2119). @AdityaSripal maybe can you confirm?

Reply:
> Connections, once established, are responsible for facilitating all cross-chain verification of IBC state

Yea that's right.

`MAY` implied optional as mentioned in the RFC, but i don't believe we stuck to a very strict technical writing standard so I defer to experts on the best way to convey this.

Reply:
I'll use the gerund verb facilitating as recommended, good catch @damiannolan but then we need plural "verifications"